sochain/README.md

```diff
@@ -0,0 +1,164 @@
+# Chainlink External Adapters to query address balance from SoChain
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+- `API_TIMEOUT`: Optional timeout param, defaults to `30000`
+
+## Input Params
+
+- `endpoint`: Optional endpoint param, defaults to `balance`
+
+### Balance endpoint
+
+- `dataPath`: Optional path where to find the addresses array, defaults to `addresses`
+- `confirmations`: Optional confirmations param, defaults to `6`
+
+- `addresses`: Addresses to query
+
+  {
+
+  - `address`: Address to query
+  - `coin`: Optional currency to query, defaults to `btc`, one of `(btc|dash|doge|ltc|zec)`
+  - `chain`: Optional chain to query, defaults to `mainnet`, one of `(mainnet|testnet)`
+
+  }
+
+```json
+{
+  "id": "1",
+  "data": {
+    "addresses": [
+      {
+        "address": "3D8DJLwUXFfZvE8yJRu729MZ8uLy25SuLz",
+        "coin": "btc"
+      },
+      {
+        "address": "3EyjZ6CtEZEKyc719NZMyWaJpJG5jsVJL1",
+        "coin": "btc"
+      },
+      {
+        "address": "38bzm6nhQMFJe71jJw1U7CbgNrVNpkonZF",
+        "coin": "btc"
+      },
+      {
+        "address": "3ANaBZ6odMrzdg9xifgRNxAUFUxnReesws",
+        "coin": "btc"
+      },
+      {
+        "address": "3FFgKaYkEf1M73QtzuY9DGqC7VeM2sAQhT",
+        "coin": "btc"
+      },
+      {
+        "address": "3KTeq879YjzhqkAXzZmdapJAVC6qz5qEth",
+        "coin": "btc"
+      },
+      {
+        "address": "35ULMyVnFoYaPaMxwHTRmaGdABpAThM4QR",
+        "coin": "btc"
+      }
+    ],
+    "confirmations": 3
+  }
+}
+```
+
+### Output
+
+```json
+{
+  "jobRunID": "1",
+  "data": {
+    "result": [
```

Comment:
If we now return all the `responses`, please add the example here. But this adds a lot of data for the node to process, I don't think this should be a default option. Maybe we should add a flag that will enable this verbose output.